src/adapt/vtree.js

```diff
@@ -672,7 +674,8 @@ adapt.vtree.newNodePositionFromNodeContext = function(nodeContext) {
         shadowType: adapt.vtree.ShadowType.NONE,
         shadowContext: nodeContext.shadowContext,
         nodeShadow: null,
-        shadowSibling: null
+        shadowSibling: null,
+        fragmentIndex: nodeContext.fragmentIndex
```

Comment:
`fragmentIndex` should always be 0 here, since this method (`newNodePositionFromNodeContext`) is used to create a `NodePosition` of the start of `PseudoColumn` (table cells, page floats or repeatable header/footer).